Lib/test/test_tkinter/test_widgets.py

```diff
@@ -7,9 +7,12 @@
 from test.test_tkinter.support import (requires_tk, tk_version,
                                   get_tk_patchlevel, widget_eq,
                                   AbstractDefaultRootTest)
+
 from test.test_tkinter.widget_tests import (
-    add_standard_options,
-    AbstractWidgetTest, StandardOptionsTests, IntegerSizeTests, PixelSizeTests)
+    add_configure_tests,
+    AbstractWidgetTest,
+    StandardOptionsTests,
+    IntegerSizeTests, PixelSizeTests)
```

Comment:
Nit: let's not stop here :)

```suggestion
    IntegerSizeTests,
    PixelSizeTests,
)
```

Reply:
OK